srcpkgs/apcupsd/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'apcupsd'
+pkgname=apcupsd
+version=3.14.14
+revision=1
+build_style=gnu-configure
+configure_args="
+ --sysconfdir=/etc/apcupsd
+ --with-pwrfail-dir=/etc/apcupsd
+ --with-upstype=usb --with-upscable=usb --enable-usb --with-dev=
+ --enable-modbus-usb
+ --enable-net --enable-pcnet
+ --enable-snmp
+ --disable-gapcmon
+ --with-nis-port=3551"
+hostmakedepends="autoconf automake"
+makedepends="libusb-compat libusb-compat-devel net-snmp net-snmp-devel man-pages man-pages-devel"
+depends="libusb-compat net-snmp nullmailer"
+conf_files="
+	etc/apcupsd/apcupsd.conf
+	etc/apcupsd/hosts.conf
+	etc/apcupsd/multimon.conf
+	etc/apcupsd/apcupsd.css
+	etc/apcupsd/changeme
+	etc/apcupsd/commfailure
+	etc/apcupsd/commok
+	etc/apcupsd/offbattery
+	etc/apcupsd/onbattery
+	etc/apcupsd/apccontrol"
+short_desc="APC UPS daemon with integrated tcp/ip remote shutdown"
+maintainer="Case_Of <voidlinux@caseof.fr>"
+license="GPL-2.0-only"
+homepage="http://www.apcupsd.org/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum="db7748559b6b4c3784f9856561ef6ac6199ef7bd019b3edcd7e0a647bf8f9867"
+
+pre_configure() {
+	export SHUTDOWN=/usr/bin/shutdown
+	export WALL=/usr/bin/wall
+	export APCUPSD_MAIL=/usr/bin/mail
+}
+
+do_install() {
+	for bin_file in apcaccess apcupsd apctest smtp; do
+		vbin src/${bin_file}
+	done
+	vmkdir etc/${pkgname}
+	vinstall platforms/apccontrol 744 etc/${pkgname}
+	for conf_file in apcupsd.conf hosts.conf multimon.conf; do
+		vinstall platforms/etc/${conf_file} 644 etc/${pkgname}
+	done
+	for conf_file in changeme commfailure commok offbattery onbattery; do
+		vinstall platforms/etc/${conf_file} 744 etc/${pkgname}
```

Comment:
What's up with these permissions?

Reply:
These scripts are only meant to be executed as root so…